### PR TITLE
Seed database

### PR DESCRIPTION
**Pull request by kendall-butt**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+const db = require('./../db/index.js');
+
+app.use('/static', express.static(path.join(__dirname, '/dist')))
+app.use(express.json())
+app.use(express.urlencoded({ extended: false }))
+
+app.get('/seller/info/', (req, res) => {
+  pool.query('SELECT * FROM seller_info', (err, data) => {
+    if (err) console.log(err)
+    res.status(200).send(data)
```
Comment (sunitmody) [inline server/index.js:16]: Does this need an else statement? If there is an error does it still send a res code of 200?

**Review by sunitmody — COMMENTED** [review]
Take a look at my comment.

File: server/index.js (hunk 1 above)
Comment (kendall-butt) [reply server/index.js]: fixed! ty

**Review by sunitmody — COMMENTED** [review]
Looks good!

**Pull request merged**